<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_book_win</class>
 <widget class="QMainWindow" name="add_book_win">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YÖNETİCİ PANELİ - KİTAP EKLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/logos.qrc">
    <normaloff>:/marun_logo/img/marun.png</normaloff>:/marun_logo/img/marun.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/logos.qrc">:/book/img/book.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>461</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>26</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Kitap Adı :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="book_name_input"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Denetleyici :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="supervisor_input"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ders : </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lecture_input"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="add_file_btn">
       <property name="text">
        <string>Dosya Ekle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="file_path_lbl">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>26</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(164, 0, 0);</string>
         </property>
         <property name="text">
          <string>İPTAL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ok_btn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(78, 154, 6);</string>
         </property>
         <property name="text">
          <string>ONAYLA</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>573</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 63 14pt &quot;URW Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Belge Kriptolama Sistemi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>341</width>
         <height>84</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>341</width>
         <height>91</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/logos.qrc">:/tf_logo/img/tf.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../res/logos.qrc"/>
  <include location="../res/logos.qrc"/>
  <include location="../res/logos.qrc"/>
 </resources>
 <connections/>
</ui>
